.teamcity/Gradle_Check/model/PerformanceTestBucketProvider.kt

```diff
@@ -305,17 +305,15 @@ echo "Performance tests to be ${action}d in this build"
 cat $performanceTestSplitDirectoryName/$action-$fileNamePostfix
 """
 
-    val linesWithEcho = csvLines.joinToString("\n") { "echo $it" }
+    val linesWithEcho = csvLines.joinToString("\n") { """echo $it >> $performanceTestSplitDirectoryName\$action-$fileNamePostfix""" }
```

Comment:
Not sure how important this is but it would probably be nice to instead properly escape the quotes here in both `$it` and in the file name.

Reply:
There are no quotes to escape. The problem here was that some test names contained `)`, and that didn't work with the `()` command on windows which allows executing multiple commands. I could escape the `)`, though it seemed easier to use `>>` instead.